spec/workers/feeds/import_articles_worker_spec.rb

```diff
@@ -2,45 +2,39 @@
 
 RSpec.describe Feeds::ImportArticlesWorker, type: :worker, sidekiq: :inline do
   let(:worker) { subject }
+  let(:feed_url) { "https://medium.com/feed/@vaidehijoshi" }
 
   include_examples "#enqueues_on_correct_queue", "medium_priority"
 
   describe "#perform" do
-    it "calls the Feeds::Import defaulting to 4 hours ago" do
+    it "processes jobs for users with feeds", vcr: "feeds_import" do
```

Comment:
the vcr cassette is needed because setting the feed_url triggers a validation (so I want to make sure we can fetch the feed, even if we stub out Import.call).

Reply:
(removed - thanks jgaskins)